### ISSUE TITLE
The note should be removed for 'satisfies a success criterion'

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#416]
### Currently

The glossary definition of 'satisfies a success criterion' has this note:

> Though WCAG2ICT and WCAG 2 don't use this exact phrase, in this document there are variations of the phrase that use this definition. See "success criteria is satisfied" in [Section 6 Comments on Conformance](https://www.w3.org/TR/wcag2ict-22/#comments-on-conformance) and "satisfy any success criterion" in the notes for the definition of [set of software programs](https://www.w3.org/TR/wcag2ict-22/#set-of-software-programs).

Issue: It's splitting hairs to say the documents don't use the exact phrase. Without a special note, other definitions are understood to apply even if the match is not exact, such as "programmatically determined" and "programmatically determinable"; "purely decorative" and "pure decoration".

Issue: The two instances mentioned both link to the definition, so there's no need to say it applies there.

Issue: We now have more uses of the term.

- "satisfy this success criterion" in an added note for 1.4.10 Reflow
- "satisfy the success criterion" in an added note for 2.1.1 Keyboard
- "this success criterion would be satisfied" in an added note for 2.1.2 No Keyboard Trap
- "satisfy this criterion" in a WCAG note for 3.3.8 Accessible Authentication
- "This success criterion would be satisfied" in an added note for 4.1.1 Parsing
- (name of SC) "would be satisfied" in an added example for 4.1.1 Parsing
- "satisfy the success criterion" in an added note for the definition of 'keyboard interface'
- In 'problematic for closed functionality':
  - "the success criterion would be satisfied" for 1.4.10 Reflow
  - "satisfy this success criterion" for 2.1.1 Keyboard
  - "this success criterion would be satisfied" for 2.1.2 No Keyboard Trap
  - "this success criterion is satisfied" for 2.1.4 Character Key Shortcuts
  - "this success criterion would be satisfied" for 2.4.7 Focus Visible

### Proposed

Remove the added note from the glossary definition of 'satisfies a success criterion'.

**Comment by bruce-usab** [comment]
I agree the note is splitting hairs.  I think it a little better to have something.  Maybe delete the leading "Though WCAG2ICT and WCAG 2 don't use this exact phrase"?  Or just affirm that the concept and term is appropriate for WCAG2ICT?

**Comment by maryjom** [comment]
The TF agreed to remove the note in the 11 July meeting. This is implemented in PR #440 and will be merged once the PR is reviewed by the editors.

**Comment by maryjom** [comment]
This issue is being closed since PR #440 has been merged.
